tests/system/test_dbapi.py

```diff
@@ -1196,7 +1232,7 @@ def test_ddl_executemany_autocommit_false(self, dbapi_database):
         table = dbapi_database.table("DdlExecuteManyAutocommit")
         assert table.exists() is False
 
-    def test_ddl_execute(self, dbapi_database):
+    def test_ddl_then_non_ddl_execute(self, dbapi_database):
         """Check that DDL statement followed by non-DDL execute statement in
         non autocommit mode results in successful DDL statement execution."""
```

Comment:
Also update the comments of these test cases, that they require `buffer_ddl=True`.

Reply:
Done